lib/libvarnishapi/vut.c

```diff
@@ -75,8 +75,12 @@ static void
 vut_vpf_remove(void)
 {
 	if (pfh != NULL) {
-		AZ(VPF_Remove(pfh));
-		pfh = NULL;
+		errno = 0;
+		if (VPF_Remove(pfh) != 0)
+			fprintf(stderr, "Cannot remove pid file: %s\n",
+				strerror(errno));
+		else
+			pfh = NULL;
```

Comment:
Always set `pfh` to NULL, if we failed to remove it once we will likely not succeed when the function is called again (because of `atexit`).